ext/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'kotlin'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```

Comment:
I think using 1.8 will be better.

Reply:
So far other module are stilling using 1.7 how able let's keep it and discuss later?

Reply:
Sure! It's cool!